data/hakki2022onset/hakki2022onset.yaml

```diff
@@ -0,0 +1,2478 @@
+# yaml-language-server:$schema=../.schema.yaml
+title: 'Onset and window of SARS-CoV-2 infectiousness and temporal correlation with
+  symptom onset: a prospective, longitudinal, community cohort study'
+doi: 10.1016/S2213-2600(22)00226-0
+description: >
+  Through a prospective, longitudinal, community cohort study that captures the critical
+  growth phase and peak of viral replication, the goal is to characterize the window
+  of SARS-CoV-2 infectiousness and its temporal relationship with symptom onset.
+analytes:
+  nose_and_throat_swab:
+    description: >
+      This analyte represents the detection and quantification of SARS-CoV-2 viral
+      RNA from throat and nose swabs specimens collected from participants. The analysis
+      focuses on measuring the viral load expressed in log10 copies/mL, with the primary
+      reference event being the onset of symptoms.
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
+    specimen: nose_and_throat_swab
```

Comment:
I think we can use a combined specimen type here without needing to add another one to the schema. There's an example here.

https://github.com/shedding-hub/shedding-hub/blob/a0bb823ab633a3136b38e1098737a21d3e8c22cb/data/han2020sequential/han2020sequential.yaml#L31-L33